app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -346,6 +348,11 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
                 view = view
             )
 
+            val shouldForwardToThirdParties =
+                PreferenceManager.getDefaultSharedPreferences(context).getBoolean(
+                    context.getPreferenceKey(R.string.pref_key_external_download_manager), false
```

Comment:
Can this be attached to the `Settings` class instead? Is it possible to use it with the Preferences UI that way?

Reply:
Unfortunately, I tried with `Settings` but it is pointing to another xml file see https://github.com/mozilla-mobile/fenix/issues/13378 where I described the issue.